app/models/manageiq/providers/vmware/infra_manager/refresher.rb

```diff
@@ -5,10 +5,9 @@ module ManageIQ::Providers
   module Vmware
     class InfraManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       def refresh
-        collector_klass = ManageIQ::Providers::Vmware::InfraManager::Inventory::Collector
-
         ems_by_ems_id.each do |_ems_id, ems|
-          collector = collector_klass.new(ems)
+          saver     = ems.class::Inventory::Saver.new(:threaded => false)
```

Comment:
why is this one threaded false?

Reply:
So this is only used for the `EmsRefresh.refresh(ems)` aka simple `rails c` case and only does a full refresh so we don't have the possibility of "falling behind" updates due to saving inventory taking time that we have with the normal RefreshWorker case.  Starting up a thread to do a single save inventory then shutting it down seemed like overkill but we can do that if you want

Reply:
No, that's fine, it was just surprising that non-spec code had threaded set to false.  Maybe it would be helpful to raise a NotImplementedError exception at the beginning of this method if in production mode? Similar to:

https://github.com/ManageIQ/manageiq/blob/647e640a075b45311e0dbac6edc64916a0ed53f8/lib/vmdb/console_methods.rb#L29

Reply:
:+1: yeah I'll add a comment over the method also